### ISSUE TITLE
Show details for failed analyses

### ISSUE THREAD

**Issue by flekschas** [parklab/refinery-platform#672]
It would be very helpful to display some details about failed analyses if possible. A protocol like Travis-CI provides could be useful to figure out at which step the analysis failed, e.g. Galaxy is down, Galaxy as an issue, Download failed...

**Comment by scottx611x** [comment]
Bioblend may be able to pull out stderr of a failed history before it gets purged

**Comment by mccalluc** [comment]
@scottx611x : Is this the same as the work we were doing earlier to get more information about failed analyses? Close?

**Comment by scottx611x** [comment]
Nope, I think you're thinking of error messages form ISA-Tab importing from here: #2036
